lineage/states/stateCommon.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, startT, endT):
         self.endT = endT
 
 
-def assign_times(lineageObj):
+def assign_times(lineageObj, *kwargs):
```

Comment:
I think ultimately, this `assign_times` function should be `StateDistribution` specific. @scottdtaylor95 had some trouble with it when he only had Gaussian observations. We can do this on a later branch though.

Reply:
See #335 (can be fixed on a later branch).